File: js/00-introduction/005-longest-word.js
function longestWord(sentence) {
    let max_word = "";
    let words = sentence.split(" ");

    for (word of words) {
        if (word.length >= max_word.length) {
            max_word = word;
        };
    };

    return max_word;
}

console.log(longestWord("what a wonderful world"));
console.log(longestWord("have a nice day"));
console.log(longestWord("the quick brown fox jumped over the lazy dog"));
console.log(longestWord("who did eat the ham"));
console.log(longestWord("potato"));

File: py/00-introduction/005-longest-word.py
def longest_word(sentence):
    max_word = ""    
    words = sentence.split(" ")

    for word in words:
        if len(word) >= len(max_word):
            max_word = word
    
    print(max_word)
    return max_word

longest_word("what a wonderful world") # -> "wonderful"
longest_word("have a nice day") # -> "nice"
longest_word("the quick brown fox jumped over the lazy dog") # -> "jumped"
longest_word("who did eat the ham") # -> "ham"
longest_word("potato") # -> "potato"
